src/renderer/src/pages/home/search-results.tsx

```diff
@@ -16,6 +20,8 @@ import { useNavigate, useSearchParams } from "react-router-dom";
 import * as styles from "./home.css";
 import { buildGameDetailsPath } from "@renderer/helpers";
 
+const PAGE_SIZE = 12;
```

Comment:
why set the pagesize here if you already set the default on the function itself to 12?

Reply:
I made it like this to increase the code readability and avoid hard coding magic numbers in the code while calculating whether this is the last results page or not

For example if a new contributor saw this line
```ts
setIsEnd(results.length < 12)
```
He will be confused and wonder what does this number  12  mean and why we put it here, right? But by declaring this constant and use it properly in the query it is clearer and will make since to him

Reply:
but you can fet this number from the search params the same way you does to get the page no?